R/scoreData.r

```diff
@@ -8,6 +8,9 @@ scoreData <- function(
     verbose = FALSE,
     ... # for the deprecated units argument
 ) {
+  lapply(c(unitrecodings, subunits), checkmate::assert_vector)
```

Comment:
I wasn't sure whether the arguments have to have a specific type

Reply:
both are data.frames